style.css

```diff
@@ -0,0 +1,253 @@
+/* CONTAINER - it's a grid*/
+
+#container {
+    position: relative;
+    background: linear-gradient();
+    padding: 0px;
+    margin: 0px;
+    height: fit-content;
+    display: grid;
+    grid-template-columns: 2fr 2fr 2fr 2fr;
+    grid-template-rows: .5fr 1fr 1fr 1fr;
+    grid-gap: 1px;
+    grid-template-areas: 
+    "header header header header"
+    "features features features features"
+    "main main main sidebar"
+    "main main main sidebar"
+    "footer footer footer footer";
+}
+
+
+/* Header sections - handling text of the banner*/
+.header {
+    grid-area: header;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100px;
+}
+.headrln {
+    width: 65vw
+}
+h1 {
+   color: orangered; 
+   text-shadow: 2px 2px 2px darkgray;
+}
+.subhead {
+    font-size: 13px;
+    text-align: center;
+}
```

Comment:
This is a potential spot where you can reuse some CSS if you know that your headers are always going to be laid out exactly the same.